types/three/examples/jsm/renderers/common/Renderer.d.ts

```diff
@@ -214,4 +229,25 @@ export default class Renderer {
     setRenderObjectFunction(renderObjectFunction: () => {}): void;
 
     getRenderObjectFunction(): () => {};
+
+    /**
+     * Runs a compute pipeline
+     */
+    compute(computeNodes: ComputeNode | ComputeNode[]): Promise<void>;
+
+    hasFeature(name: string): boolean;
+
+    copyFramebufferToTexture(framebufferTexture: FramebufferTexture): void;
+
+    readRenderTargetPixelsAsync(renderTarget: RenderTarget, x: number, y: number, width: number, height: number): void;
```

Comment:
Seems like this should return a promise that can return some sort of typed array?

Reply:
Oh good catch. will take a look